spec/lib/salus/scanners/brakeman_spec.rb

```diff
@@ -35,6 +35,26 @@
         expect(logs).to include('Dangerous Eval')
       end
 
+      it 'should respect the config for ignoring files' do
```

Comment:
there seems like there should be more tests for all the config options.

Writing the integration tests are a pita but help us make sure the underlying scanners don't break underneath us when doing an upgrade.